### PR TITLE
feat: simple map

### LINKED ISSUE #2: Implement map
Implement a fullscreen map

### PR DESCRIPTION
**Pull request by YannRostingt**
mapr

close #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,19 @@
...
+    "google-map-react": "^1.1.4",
+    "google-maps-react": "^2.0.2",
+    "leaflet": "^1.5.1",
     "@material-ui/core": "^4.2.0",
     "@material-ui/icons": "^4.2.1",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-google-map": "^3.1.1",
```
Comment (LucasLeRay) [inline package.json]: Not necessary anymore

File: src/Components/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const Map = ReactMapboxGl({
+  accessToken:
+    'pk.eyJ1IjoiZmFrZXVzZXJnaXRodWIiLCJhIjoiY2pwOGlneGI4MDNnaDN1c2J0eW5zb2ZiNyJ9.mALv0tCpbYUPtzT7YysA2g',
+})
+
+export default function SimpleMap() {
+  return (
+    <div>
+      <Map
+        center={[5.36978, 43.296482]}
```
Comment (LucasLeRay) [inline src/Components/Map/Map.js]: Why ?

File: src/Components/Map/Map.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        style="mapbox://styles/mapbox/streets-v9" // eslint-disable-line
+        containerStyle={{
+          height: '1080px',
```
Comment (LucasLeRay) [inline src/Components/Map/Map.js]: 🤔

File: src/Components/Map/Map.js (hunk 2 above)
Comment (YannRostingt) [reply src/Components/Map/Map.js]: After, i will replace that by the user geolocation

File: src/components/Map/Map.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import ReactMapboxGl from 'react-mapbox-gl'
+
+const Map = ReactMapboxGl({
+  accessToken:
+    'pk.eyJ1IjoiZmFrZXVzZXJnaXRodWIiLCJhIjoiY2pwOGlneGI4MDNnaDN1c2J0eW5zb2ZiNyJ9.mALv0tCpbYUPtzT7YysA2g',
```
Comment (LucasLeRay) [inline src/components/Map/Map.js]: No API KEY in frontend

**Pull request merged**